### DISCUSSION TITLE
Snippet with regex transformation

### DISCUSSION THREAD

**Discussion by sangramsingha** [saghen/blink.cmp#727] [Q&A] [answered]
I have some custom snippet which I use for Java.
For java package (name space) but they don't seem to be supported yet by `vim.snippet`

Example 
snippet : `"package ${TM_DIRECTORY/.+java\\/|([^\\/]+)|(\\/)/$1${2:+.}/g};"`
file path :  `/home/foo/bar/src/main/java/com/example/Main.java `
should result in : `package com.example`

With nvim-cmp, the above snippet work fine by using
```lua
{
    "L3MON4D3/LuaSnip",
    version = "v1.2.*",
    build = "make install_jsregexp",
}
```

Is there a way to include `L3MON4D3/LuaSnip` in `blink-cmp` for regex transformation of snippet variables.

**Comment by saghen** [comment, marked as answer]
https://cmp.saghen.dev/configuration/snippets.html#luasnip

**Reply by sangramsingha** [reply]
Awesome!!

My bad, should have read the docs thoroughly.
